pkg/operator/options/options.go

```diff
@@ -29,19 +30,20 @@ import (
 	"github.com/aws/karpenter-core/pkg/utils/env"
 )
 
-var validLogLevels = []string{"", "debug", "info", "error"}
+var (
```

Comment:
Maybe I'm missing it but where does the global flagset get defined?

Reply:
I didn't actually use a global flagset since it's scope could be limited to `injection.WithOptionsOrDie`. Do you still think there's value in a global flagset?